src/extension/background-script/nostr/index.ts

```diff
@@ -69,6 +82,14 @@ class Nostr {
     return Utf8.stringify(decrypted);
   }
 
+  nip44Encrypt(peer: string, plaintext: string) {
```

Comment:
Can the existing encrypt and functions be prefixed with `nip04`?

Reply:
resolved